package.json

```
@@ -5,7 +5,7 @@
     "build": "node ./scripts/build.js",
     "clean": "git clean -fdX .",
     "start": "node ./scripts/start.js",
-    "test": "node ./scripts/test.js"
+    "test": "jest --config jest.config.js"
```

```suggestion
    "test": "jest"
```
picked up by default

---

yeah. I typically like to be verbose here to highlight exactly what is going on; avoid some of the "magic" for newer people. This is, however, a personal preference. Happy to remove